presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/MergeIdenticalWindows.java

```diff
@@ -26,11 +26,9 @@
 
```

Comment:
Is the message for this commit accurate? It just looks like a minor refactoring that removes a `SpecificationAndFrame` and replaces it with `WindowNode.Specification`